Sources/Repository.swift

```diff
@@ -20,6 +20,7 @@ public protocol Repository {
     func getElements(filteredBy predicateFormat: String, _ args: Any..., completion: (AnyCollection<Model>) -> Void)
     func getElements(filteredBy predicate: NSPredicate, completion: (AnyCollection<Model>) -> Void)
     func getElement<Id>(withId id: Id, _ completion: (Model?) -> Void)
+    func getElements(sorted keyPath: String, ascending: Bool, completion: (AnyCollection<Model>) -> Void)
```

Comment:
Just a quick discussion: 
I have seen some implementations that use enums for this like 

``` ASCENDING, DESCENDING ``` 

I am fine with both approaches, but I would favor one for readability. 

@JulianBissekkou @grafele Your opinions?